### ISSUE TITLE
[Android] App permissions access makes the VPN to crash

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla-mobile/mozilla-vpn-client#1335]
#### Affected versions:
- 2.4.0 (16262900) debug build;

#### Affected Platforms:
- Android 10;

#### Prerequisites:
- Have the VPN app installed;
- Have a FxA account with subscription;

#### Steps to reproduce:
1. Open the VPN app;
2. Tap “Get Started”;
3. Finalize authentication;
4. Go to “Settings”;
5. Tap “App permissions”;
6. Wait few seconds;

#### Expected result:
-  The option can be accessed.

#### Actual result:
- The VPN app crashes.

#### Notes:
- Attaching the logs.
[yykP6Tmc.txt](https://github.com/mozilla-mobile/mozilla-vpn-client/files/6828882/yykP6Tmc.txt)

**Comment by bakulf** [comment]
```
07-16 17:44:57.383  5419  5419 F DEBUG   : *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
07-16 17:44:57.384  5419  5419 F DEBUG   : Build fingerprint: 'HUAWEI/PRA-LX1/HWPRA-H:8.0.0/HUAWEIPRA-LX1/407(C432):user/release-keys'
07-16 17:44:57.384  5419  5419 F DEBUG   : Revision: '0'
07-16 17:44:57.384  5419  5419 F DEBUG   : ABI: 'arm64'
07-16 17:44:57.384  5419  5419 F DEBUG   : pid: 5312, tid: 5411, name: QtThread  >>> org.mozilla.firefox.vpn.debug:QtOnlyProcess <<<
07-16 17:44:57.384  5419  5419 F DEBUG   : signal 6 (SIGABRT), code -6 (SI_TKILL), fault addr --------
07-16 17:44:57.388  5419  5419 F DEBUG   : Abort message: 'java_vm_ext.cc:504] JNI DETECTED ERROR IN APPLICATION: java_class == null'
07-16 17:44:57.388  5419  5419 F DEBUG   :     x0   0000000000000000  x1   0000000000001523  x2   0000000000000006  x3   0000000000000008
07-16 17:44:57.388  5419  5419 F DEBUG   :     x4   0000000000000000  x5   0000000000000000  x6   0000000000000000  x7   7f7f7f7f7f7f7f7f
07-16 17:44:57.388  5419  5419 F DEBUG   :     x8   0000000000000083  x9   776689fe97893e85  x10  0000000000000000  x11  0000000000000001
07-16 17:44:57.388  5419  5419 F DEBUG   :     x12  ffffffffffffffff  x13  ffffffffffffffff  x14  ff00000000000000  x15  ffffffffffffffff
07-16 17:44:57.388  5419  5419 F DEBUG   :     x16  0000007aac9d92d0  x17  0000007aac97aefc  x18  0000000000000048  x19  00000000000014c0
07-16 17:44:57.388  5419  5419 F DEBUG   :     x20  0000000000001523  x21  0000007a73997f00  x22  0000000000000002  x23  00000000000006f9
07-16 17:44:57.388  5419  5419 F DEBUG   :     x24  0000000000000001  x25  0000007a7f9a6f00  x26  00000000000006f8  x27  0000007a7f501620
07-16 17:44:57.388  5419  5419 F DEBUG   :     x28  0000000000000059  x29  0000007a7f5014c0  x30  0000007aac92db94
07-16 17:44:57.388  5419  5419 F DEBUG   :     sp   0000007a7f501480  pc   0000007aac97af04  pstate 0000000060000000
07-16 17:44:57.393  5419  5419 F DEBUG   : 
07-16 17:44:57.393  5419  5419 F DEBUG   : backtrace:
07-16 17:44:57.393  5419  5419 F DEBUG   :     #00 pc 000000000006af04  /system/lib64/libc.so (tgkill+8)
07-16 17:44:57.393  5419  5419 F DEBUG   :     #01 pc 000000000001db90  /system/lib64/libc.so (abort+88)
07-16 17:44:57.393  5419  5419 F DEBUG   :     #02 pc 00000000004380d0  /system/lib64/libart.so (_ZN3art7Runtime5AbortEPKc+528)
07-16 17:44:57.393  5419  5419 F DEBUG   :     #03 pc 00000000004387e0  /system/lib64/libart.so (_ZN3art7Runtime7AborterEPKc+24)
07-16 17:44:57.393  5419  5419 F DEBUG   :     #04 pc 0000000000523e58  /system/lib64/libart.so (_ZN7android4base10LogMessageD1Ev+900)
07-16 17:44:57.393  5419  5419 F DEBUG   :     #05 pc 00000000002d5c7c  /system/lib64/libart.so (_ZN3art9JavaVMExt8JniAbortEPKcS2_+1716)
07-16 17:44:57.393  5419  5419 F DEBUG   :     #06 pc 00000000002d5f48  /system/lib64/libart.so (_ZN3art9JavaVMExt9JniAbortFEPKcS2_z+176)
07-16 17:44:57.393  5419  5419 F DEBUG   :     #07 pc 0000000000319444  /system/lib64/libart.so (_ZN3art3JNI11GetMethodIDEP7_JNIEnvP7_jclassPKcS6_+1380)
07-16 17:44:57.393  5419  5419 F DEBUG   :     #08 pc 0000000000232568  /data/app/org.mozilla.firefox.vpn.debug-wrb_9juaBC-PO8Ge6Q0oJg==/lib/arm64/libQt5Core_arm64-v8a.so
07-16 17:44:57.393  5419  5419 F DEBUG   :     #09 pc 0000000000232144  /data/app/org.mozilla.firefox.vpn.debug-wrb_9juaBC-PO8Ge6Q0oJg==/lib/arm64/libQt5Core_arm64-v8a.so
07-16 17:44:57.393  5419  5419 F DEBUG   :     #10 pc 0000000000232be4  /data/app/org.mozilla.firefox.vpn.debug-wrb_9juaBC-PO8Ge6Q0oJg==/lib/arm64/libQt5Core_arm64-v8a.so (_ZNK17QJNIObjectPrivate11callMethodVIiEET_PKcS3_St9__va_list+52)
07-16 17:44:57.393  5419  5419 F DEBUG   :     #11 pc 00000000000100e4  /data/app/org.mozilla.firefox.vpn.debug-wrb_9juaBC-PO8Ge6Q0oJg==/lib/arm64/libQt5AndroidExtras_arm64-v8a.so (_ZNK17QAndroidJniObject10callMethodIiEET_PKcS3_z+112)
```

**Comment by ValentinaPC** [comment]
Verified this as fixed on Android 10, VPN 2.4 (16269380).
The app no longer crashes while accessing the  App Permissions page.
